### ISSUE TITLE
flatpak-builder: support for perl buildsystem

### ISSUE THREAD

**Issue by nhlfr** [flatpak/flatpak#521]
CPAN packages usually are build with the following commands:

```
perl Makefile.PL
make
```

I guess we can treat `perl Makefile.PL` as a configure phase for potential `perl` buildsystem.

The good example of a module which uses such a buildsystem and would be needed by several desktop apps is Berkeley DB.

**Comment by bochecha** [comment]
> CPAN packages usually are build with the following commands:

That's true for the modules using ExtUtils::MakeMaker.

However there's also Module::Build, which uses a `Build.PL` file instead of the `Makefile.PL` one.

And then there's Module::Install, autoperl, ...

Back in the days I used to package lots of perl modules as RPMs (we all have parts of our past we'd rather forget 😜), and the amount of build systems I encountered was mind boggling. Some even required themselves to build, with no way to bootstrap them!

All in all, the Perl buildsystems situation always appeared to me to be even more "diverse" than the Python one (TIMTOWTDI!), and we're already going with a generic buildsystem (see #134) instead of a specific one for Python.
